atomic_reactor/utils/retries.py

```diff
@@ -0,0 +1,60 @@
+"""
```

Comment:
Was this move necessary? It should have also separate test_suite file then.

Reply:
It was, otherwise there would be a cyclic import between `auth` and `util` (commit message says this too). I will look for relevant tests and move them too.

Reply:
Plot twist: there were no tests, I added some basic ones (and remove the failing one in `test_auth`)